python/ray/ray_logging.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import os
+
+_default_handler = None
+
+
+def setup_logger(logging_level, logging_format):
+    """Setup default logging for ray."""
+    logger = logging.getLogger("ray")
+    if type(logging_level) is str:
+        logging_level = logging.getLevelName(logging_level.upper())
```

Comment:
what if it isn't a string? do we need to validate this?

Reply:
If this is logging.INFO kind of type, it works correctly. I think there's no need to add validation here because this method was just copied (but I'd love to add isinstance(str) and logging type if you'd like to!)